2024-08-10 05:32:44.564 INFO page_ 1:conftest.py:34: new page opened: [<Page url='about:blank'>]
2024-08-10 05:32:44.570 DEBUG tests_api.utils.logger:logger.py:23: > GET https://dev.agentum.pro/backend/api/question/questions-for-user {'method': 'GET', 'url': 'https://dev.agentum.pro/backend/api/question/questions-for-user', 'params': None, 'headers': {'Content-type': 'application/json', 'Authorization': 'Bearer YwQogs0aDUHzYIgAIN88OBaTDiKesDmAdciD9vkz'}}
2024-08-10 05:32:44.571 DEBUG urllib3.connectionpool:connectionpool.py:1048: Starting new HTTPS connection (1): dev.agentum.pro:443
2024-08-10 05:32:44.665 DEBUG urllib3.connectionpool:connectionpool.py:546: https://dev.agentum.pro:443 "GET /backend/api/question/questions-for-user HTTP/1.1" 502 157
2024-08-10 05:32:44.666 DEBUG tests_api.utils.logger:logger.py:27: < 502 https://dev.agentum.pro/backend/api/question/questions-for-user <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.18.0</center>
</body>
</html>

2024-08-10 05:32:44.666 DEBUG tests_api.utils.logger:logger.py:35: curl -X GET -H 'Accept: */*' -H 'Accept-Encoding: gzip, deflate' -H 'Authorization: Bearer YwQogs0aDUHzYIgAIN88OBaTDiKesDmAdciD9vkz' -H 'Connection: keep-alive' -H 'Content-type: application/json' -H 'User-Agent: python-requests/2.31.0' https://dev.agentum.pro/backend/api/question/questions-for-user